src/GuestConfiguration/GuestConfiguration/ChangeLog.md

```diff
@@ -19,6 +19,10 @@
 -->
 ## Upcoming Release
 
+## Version 0.10.3
```

Comment:
@aashish1987 same comment here, you can remove this header and put the below two lines under `## Upcoming Release` since we'll do versioning before the module is released